### PR TITLE
added buy page

### PR DESCRIPTION
**Pull request by 1ShahidBashir**
# Pull Request Format
## Type of PR
- **Add X in the box to specify the improvement type.**
- [] Bug fix
- [x ] Feature enhancement
- [ ] Documentation update
- [ ] Other (specify): ___________  

## Description
This pull request aims to [briefly describe what the pull request does, e.g., "add a new feature," "fix a bug," "improve documentation," etc.]. 

## Screenshots / Videos (if applicable)
**Before:**
- [Describe the state before the changes, e.g., "The section headings in the footer were misaligned, leading to an inconsistent and cluttered appearance."]

**After:**
- [Describe the state after the changes, e.g., "The section headings in the footer are now properly aligned, providing a clean and professional look."]

## Checklist
- **Add X in the box to specify.**
- [X] I have performed a self-review of my code.
- [x ] I have tested the changes thoroughly before submitting this pull request.
- [ x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [ x] I have commented my code, particularly in hard-to-understand areas.

### Additional Context
[Provide any additional context about the changes, such as specific challenges you faced or decisions you made. This can help reviewers understand the rationale behind your changes.]
image

Thank you for reviewing my pull request!

### CODE REVIEW

**Comment by 1ShahidBashir** [comment]
@swaraj-das please check this pr

**Comment by 1ShahidBashir** [comment]
@swaraj-das  please let me know if it is good

**Comment by swaraj-das** [comment]
@1ShahidBashir You should give the issue number.

**Comment by 1ShahidBashir** [comment]
@swaraj-das 253

**Comment by 1ShahidBashir** [comment]
@swaraj-das Please Check the PR now

File: buy.html [hunk 1]
```diff
@@ -0,0 +1,260 @@
+<style>
```
Comment (swaraj-das) [inline buy.html:1]: Make another folder named "Buy" and don't add css in html file. Make html and css file separately.

Comment (1ShahidBashir) [reply buy.html]: done

**Comment by 1ShahidBashir** [comment]
@swaraj-das I have put the files in a separate folder named buy.
Please check it out

**Comment by 1ShahidBashir** [comment]
@swaraj-das Please check the pr

**Comment by 1ShahidBashir** [comment]
@swaraj-das It has been a long time please check this PR !!!

**Comment by 1ShahidBashir** [comment]
@swaraj-das check this pr

**Pull request closed without merging**